core/src/main/scala/com/databricks/labs/remorph/intermediate/extensions.scala

```diff
@@ -136,6 +136,14 @@ case class Timezone(expression: Expression, timeZone: Expression) extends Binary
 
 case class WithinGroup(expression: Expression, order: Seq[SortOrder]) extends Unary(expression) {
   override def dataType: DataType = expression.dataType
+
+  override def makeCopy(newArgs: Array[AnyRef]): Expression = {
```

Comment:
`TreeNode.makeCopy` docs say:
```
Must be overridden by child classes that have
   * constructor arguments that are not present in the productIterator.

```
I don't know exactly why only 1 arg of the `WithinArg` ctor is seen by the `productIterator` (maybe because they are not `val`s ?), but it's what happens.

This is tested via existing test cases in `SnowflakeToDatabricksTranspilerTest`, which crash without this bug fix.